packages/react/src/avatar/Avatar.stories.tsx

```diff
@@ -1,28 +1,29 @@
-import { ComponentStory, ComponentMeta } from '@storybook/react';
-import { Flex, Stack } from '../box';
+import { Meta, StoryObj } from '@storybook/react';
+import { Flex } from '../box';
 import { boxPalette } from '../core';
 import { Text } from '../text';
 import { Avatar } from './Avatar';
 
-export default {
+const meta: Meta<typeof Avatar> = {
 	title: 'Content/Avatar',
 	component: Avatar,
-} as ComponentMeta<typeof Avatar>;
+	args: {
+		name: 'William Mead',
+		tone: 'neutral',
+		size: 'md',
+	},
+};
 
-const Template: ComponentStory<typeof Avatar> = (args) => <Avatar {...args} />;
+export default meta;
 
-export const Basic = Template.bind({});
-Basic.args = {
-	name: 'William Mead',
-	tone: 'neutral',
-	size: 'md',
-};
+type Story = StoryObj<typeof Avatar>;
 
-export const Tone = Template.bind({});
-Tone.args = {
-	name: 'William Mead',
-	tone: 'action',
-	size: 'md',
+export const Neutral: Story = {};
```

Comment:
The first story is now 'neutral' rather than basic. This is reflected in the markdown file below